webserver/tests/Makefile

```
@@ -65,11 +63,9 @@ run_test: venv_dev
 		nosetests $(NOSECFG) $(ROOT)/server/${TEST} \
 		&& { ${SHUTDOWN_SERVER_CMD}; } || ${EXIT_ERROR}
 
-test_unit: venv_dev thrift
-	$(ACTIVATE_DEV_VENV) && $(UNIT_TEST_CMD)
+test_unit: test_unit_server
 
-test_unit_novenv: thrift
-	$(UNIT_TEST_CMD)
+test_unit_novenv: unit_novenv_server
```

Shouldn't the test target start with `test`?
```suggestion
test_unit_novenv: test_unit_novenv_server
```